source/includes/_content_access_management.md

```diff
@@ -0,0 +1,283 @@
+
+Access to a given content item can be defined at the following levels:
+- User based
+- Group Based
+- Group and Role based
+
+It's important to note that you you can create multiple access records tying content items to multiple 
+The ability to edit the content item can be granted by passing true for the `canEdit` flag. 
+
+## Get Access Records associated to a repository
+
+> Access Records Response
+
+```json
+{
+    "data": [
+        {
+            "contentItemId": "exampleId1",
+            "userId": "seattle_user_1",
+            "groupId": "seattle_office",
+            "canEdit": false,
+            "role": "group_user",
+            "isDeleted": false
+        }, // more records if applicable
+    ],
+    "nextPageToken": "next_page_token", // only present when there is another page
+    "previousPageToken": "previous_page_token" // only present when there is a previous page
+}   
+```
+
+If you decline to pass query params - this call will return all access records associated with a given repository. Otherwise it will return return a subset based on the provied query parameters.
+
+**Response Codes**:
+
+- `200 OK`
+
+### HTTP Request
+
+`GET management/v1/content/{repositorySlug}`
+
+### URL Params
+
+| URL Param | Type   | Required  | Description                                    |
+| --------- | ------ | --------- | ---------------------------------------------- |
+| repositorySlug | String | true | The slug of the content repository whose access records you wish to view. |
+
+### Request Query Params
+
+| URL Param | Type   | Required | Description                                    |
+| --------- | ------ | -------- | ---------------------------------------------- |
+| userId | string | false | Query for all records that match the provided userId. | 
+| groupId | string | false | Query for all records that match the provided groupId, note that this will return records that may or may not have a userId associated with them. |
+| role | string | false | Query for all records that match the provided role. If this is omitted, all records will be return according to the other criteria provided. |  
+
+
+## Get Access Records associated to a repository and content item
+
+> Access Records Response
+
+```json
+{
+    "data": [
+        {
+            "contentItemId": "exampleId1",
+            "userId": "seattle_user_1",
+            "groupId": "seattle_office",
+            "canEdit": false,
+            "role": "group_user",
+            "isDeleted": false
+        }, // more records if applicable
+    ],
+    "nextPageToken": "next_page_token", // only present when there is another page
+    "previousPageToken": "previous_page_token" // only present when there is a previous page
+}   
+```
+
+### HTTP Request
+
+`GET management/v1/content/{repositorySlug}/{contentItemId}`
```

Comment:
And same for these, I think this should be `management/v1/repository/{repositorySlug}/item/{contentItemId}`.